src/arith.js

```diff
@@ -1627,3 +1627,32 @@ CPU.prototype.saturate_uw = function(v)
     dbg_assert(v >= 0);
     return v > 0xFFFF ? 0xFFFF : v;
 };
+
+CPU.prototype.integer_round = function(f)
```

Comment:
This is from fpu.js, isn't it? Could you refactor the code in fpu.js to use this bit?

Reply:
Yes it does. I have refactored the code in *fpu.js*, with an encapsulation of *cpu.integer_round* to minimize the number of impacted lines. (But at the cost of an extra function call, which is not necessarily a good thing in terms of optimization, in my opinion.)